ckcc/client.py

```diff
@@ -90,7 +90,9 @@ def resync(self):
 
         # check the above all worked
         err = self.dev.error()
-        if err != '':
+        if err == 'hid_error is not implemented yet':
+            print("WARN: hid_error_not_implemented error thrown. Suppressing.")
+        elif err != '':
             raise RuntimeError('hidapi: '+err)
```

Comment:
I don't know what is the implementation/behavior of the `UnixSimulatorPipe`,
but when `dev` is a hid device, this check makes absolutely no sense - `dev.read(...)` raise an exception when it fails, see here: https://github.com/trezor/cython-hidapi/blob/894df92a9310559eb84942475bd74c2bd6fe13ee/hid.pyx#L123

this code would never execute

Reply:
I can't remember ever seeing this "hidapi: ..." error being thrown, although I would expect it only in cases where the Coldcard is disconnected out during operation.